### ISSUE TITLE
Sys.ssl missing for java

### ISSUE THREAD

**Issue by shintiger** [HaxeFoundation/haxe#7525]
```
package;

import java.Lib;
import sys.ssl.Socket;

/**
 * ...
 * @author Tiger
 */
class Main 
{
	
	static function main() 
	{
		var sock:Socket = new Socket();
	}
	
}
```

Works very fine with cpp and js target but java:

Compiling:
haxe -main Main -java java

Error:
src\haxe\root\Main.java:37: package sys.ssl does not exist
sys.ssl.Socket ws = new sys.ssl.Socket();

haxe:3.4.7
hxjava:3.2.0

java -version:1.6.0_45
[hxjava_build.txt](https://github.com/HaxeFoundation/haxe/files/2454093/hxjava_build.txt)

cmd:
```
src/haxe/root/Std.java
src/haxe/lang/IEquatable.java
src/haxe/lang/Exceptions.java
src/haxe/lang/HaxeException.java
src/haxe/root/Array.java
src/_Array/ArrayIterator.java
src/sys/net/Socket.java
src/haxe/lang/IHxObject.java
src/haxe/lang/HxObject.java
src/haxe/lang/DynamicObject.java
src/haxe/lang/Enum.java
src/haxe/lang/ParamEnum.java
src/haxe/lang/EmptyObject.java
src/haxe/lang/Runtime.java
src/haxe/lang/StringExt.java
src/haxe/lang/StringRefl.java
src/haxe/lang/FieldLookup.java
src/haxe/io/Eof.java
src/haxe/root/Type.java
src/haxe/root/Main.java
src/haxe/lang/Function.java
src/haxe/lang/VarArgsBase.java
src/haxe/lang/VarArgsFunction.java
src/haxe/lang/Closure.java
src/haxe/root/Reflect.java
src/haxe/root/StringBuf.java

```

**Comment by nanjizal** [comment]
nadako

https://github.com/HaxeFoundation/haxe/blob/development/std/sys/Http.hx#L92
#elseif java
sock = new java.net.SslSocket();

but it should say 
sock = new java.net.Socket();

since
https://github.com/HaxeFoundation/haxe/blob/development/std/java/_std/sys/net/Socket.hx

Original commit has this typo in.
https://github.com/HaxeFoundation/haxe/commit/5f965678b745ad2f0cecbd92c8216979272cc6db#diff-46c39ad1ca1335710313d6f143031a52R377

Reluctant to do a pull as Sockets are not an area I know much about, but maybe worth a try.

Best
Justin

**Comment by Simn** [comment]
The Java target will be put in "regression fixes only" state with the Haxe 4.3.0 release. See https://github.com/HaxeFoundation/haxe/wiki/Target-Support-Level#java-and-c-since-430 for additional information. We apologize for any inconvenience this causes!
